include/z64animation.h

```diff
@@ -99,6 +99,9 @@ typedef enum {
 
 // Enables the movement of an actor in the Y-axis based on the current animation.
 // This only has an effect if the "Actor Move" Anim Task is in use.
+// Due to how movement and gravity works, actors will appear to be pulled to the ground
+// if nearby. To avoid this undesireable snapping and stuttering, `ANIM_FLAG_OVERRIDE_MOVEMENT`
+// can be used.
```

Comment:
```c
//
// This animation-driven movement does not replace movement applied elsewhere, like from gravity, collision...
// The actor should stop updating other sources of movement as required if they prevent playing the animation as
// intended. An option is to implement and use `ANIM_FLAG_OVERRIDE_MOVEMENT`.
```
?

Reply:
what else needs to go in "..."? I cant think of anything more

Reply:
xz velocity from speed and displacement from OC colliders are two things that would apply in Player

Since we're choosing to name this not after Player though I'm staying vague as the source of movement can be whatever an actor wants to do

Reply:
alright, I modified both a bit. let me know how they are